integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -3210,7 +3206,7 @@ msgstr "tünews aktivieren"
 
 #: cms/models/regions/region.py:290
 msgid "Enable to show a feed of tunews articles to users."
-msgstr "Auswählen, um Benutzern einen tünews-Feed anzuzeigen."
+msgstr "Auswählen, um anderen Konten einen tünews-Feed anzuzeigen."
```

Comment:
```suggestion
msgstr "Auswählen, um Benutzer*innen einen tünews-Feed anzuzeigen."
```
Here as well - I think app users are meant.

Reply:
If I'm not mistaken this should be resolved now